src/scripts/run_full_dialogue_pipeline.js

```
@@ -1,221 +1,235 @@
         // Generate individual dialogue (interactive)
-        await runScript(GENERATE_SCRIPT);
-        break;
-        
+        await runScript(GENERATE_SCRIPT)
+        break
+
       case '2':
         // Batch generate dialogues
-        const options = await getBatchOptions();
-        
+        const options = await getBatchOptions()
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)